test/Faker/Provider/InternetTest.php

```diff
@@ -108,6 +108,16 @@ public function testIpv4()
         $this->assertNotFalse(filter_var($this->faker->ipv4(), FILTER_VALIDATE_IP, FILTER_FLAG_IPV4));
     }
 
+    public function testIpv4NotLocalNetwork()
+    {
+        $this->assertNotEquals('1.0.0.0', long2ip(ip2long($this->faker->ipv4()) & (-1 << (32 - 8))));
```

Comment:
@ravage84 [`assertLessThanOrEqual(0x01000000, ip2long($this->faker->ipv4())`](https://phpunit.de/manual/current/en/appendixes.assertions.html#appendixes.assertions.assertLessThanOrEqual).